package.json

```diff
@@ -44,7 +44,7 @@
     "@types/react-plotly.js": "^2.5.0",
     "@types/react-test-renderer": "^16.9.1",
     "antlr4ts-cli": "^0.5.0-alpha.4",
-    "cypress": "^6.0.0",
+    "cypress": "12.11.0",
```

Comment:
Maybe better to upgrade it in a separate PR after we have all-green status for all the tests

Reply:
There are some test failures that dont' seem to be fixable in Cypress 6.
Cypress has had significant refactors (6 Majors!).  The internal browser-automation has been changed. (Investigating 6 majors worth the code-changes is infeasible).
Some browser-automation techinques from Cypress 6 -> 12.x have changed in ways that are difficult to document.  The APIs have changed, which provides a small clue.  Other changes are simply observed and trial-and-error + intuition is necessary to determine the "correct" repeatable automation.